pkg/command/service/load.go

```diff
@@ -305,8 +295,78 @@ func getReplicasCount(loadResult pkg.LoadResult) (int, []int) {
 	return maxReplicasCount, replicasCountList
 }
 
+// runInternalLoadTool runs internal load test tool(vegeta) using library, returns load output and error
+func runInternalLoadTool(inputs pkg.LoadArgs, endpoint string, host string) (output string, err error) {
```

Comment:
Might be simpler to name this `runVegataLoadTool` or something along those lines

Reply:
No problom, it is named `runInternalLoadTool` originally to distinguish external load tool(wrk, hey), but renamed `vegeta` is easier to understand, I will address it.